crates/ruff_python_ast/ast.toml

```diff
@@ -335,8 +348,8 @@ fields = [
 [Expr.nodes.ExprIf]
 doc = "See also [IfExp](https://docs.python.org/3/library/ast.html#ast.IfExp)"
 fields = [
-  { name = "test", type = "Expr" },
   { name = "body", type = "Expr" },
+  { name = "test", type = "Expr" },
   { name = "orelse", type = "Expr" },
 ]
```

Comment:
I'd prefer to keep the field order as is because it represents the semantic order of the fields and instead override the order with `source_order`. This should also reduce the size of this diff